.gitignore

```diff
@@ -1,20 +1,29 @@
-*#
+# Python
 *.py[co]
+.virtualenv
+_venv
+.pytest_cache/
+
+# Node
+node_modules
+
+# WPT repo stuff
+/MANIFEST.json
+
+testharness_runner.html
+!/tools/wptrunner/wptrunner/testharness_runner.html
+
+_certs
+config.json
+
+# Various OS/editor specific files
+*#
 *.sw[po]
 *~
-MANIFEST.json
-!content-security-policy/support/manifest.json
-!payment-handler/manifest.json
 \#*
-_certs
-.virtualenv
-config.json
-node_modules
 scratch
```

Comment:
What even is this? Should this be in the previous section?

Reply:
No idea

Reply:
Seems to be from 8267f079550cbe16e488fa85a04d7d7d309f6529, added along with node. But Google suggests it isn't node, so who knows.